pynest/nest/__init__.py

```diff
@@ -198,6 +202,40 @@ def init(argv):
         _kernel.NESTError("Initiatization of NEST failed.")
 
 
+def expand_sli_path():
+    """ Add paths defined in SLI_PATH to searchpath. """
+
+    sli_paths = [path for path in
+                 os.environ.get("SLI_PATH", "").split(os.path.pathsep)
+                 if path != ""]
+
+    assert_sli_valid(sli_paths)
+
+    for path in sli_paths:
+        sli_run("({}) addpath".format(path))
+
+
+def assert_sli_valid(names):
+    for n in names:
+        if re.search(r"[\(\)\\]", n):
+            raise _kernel.NESTError("Path or Module names must not contain "
+                "parentheses or backslashes.")
```

Comment:
pynest/nest/__init__.py:221:17: E128 continuation line under-indented for visual indent